.eslintrc.js

```diff
@@ -9,5 +9,6 @@ module.exports = {
     browser: true
   },
   rules: {
+    'no-console': ["error", { allow: ["warn", "error"] }]
```

Comment:
Ember.Logger is deprecated in the new module structure - switched the one usage over to `console.warn`